client/src/modules/cohort/components/CohortDescription.jsx

```diff
@@ -43,10 +48,24 @@ class CohortDescription extends PureComponent {
     const {
       details: { description: previousDescription }
     } = previousProps;
+    const { descriptionInputValue } = this.state;
+    const dataHasChanged = previousDescription !== descriptionInputValue;
```

Comment:
inputValueHasChanged?

Reply:
I think data has changed is understandable enough.